plugins/seoMetaGenerator.ts

```diff
@@ -16,6 +17,25 @@ type MetaProperties = {
 }
 
 export default function ({ app }, inject): void {
+  const getMetaType = (mediaType: MediaType | string | undefined): string => {
+    let ogType: string | undefined
+    switch (mediaType) {
+      case MediaType.VIDEO:
+        ogType = 'video:other'
```

Comment:
```suggestion
        return 'video:other'
```

Reply:
since there is a break no need to return?
not really a fan of switch ... case, can we have another solution?

Reply:
if else?